index.js

```diff
@@ -1207,6 +1246,9 @@ ArrayTome.prototype.shift = function () {
 		diff(this, 'shift');
 	}
 
+	// update root
+	setRoot(out, out);
+
 	return out ? out.valueOf() : out;
```

Comment:
Why not just return `Tome.conjure(out)`?

You're going to run into trouble maintaining your setRoot version of tomes.